dashboard/src/main/java/org/apache/uniffle/dashboard/web/JettyServerFront.java

```diff
@@ -83,14 +88,32 @@ private void initialization() {
   }
 
   private void setRootServletHandler() {
-    final ContextHandler contextHandler = new ContextHandler("/");
+    HandlerList handlers = new HandlerList();
+    ResourceHandler resourceHandler = addResourceHandler();
+    String backedAddress = conf.getString(DashboardConf.DASHBOARD_BACKEND_ADDRESS);
+    ServletContextHandler servletContextHandler = addProxyHandler(backedAddress);
+    handlers.setHandlers(new Handler[] {resourceHandler, servletContextHandler});
+    server.setHandler(handlers);
+  }
+
+  private static ResourceHandler addResourceHandler() {
     ResourceHandler resourceHandler = new ResourceHandler();
     resourceHandler.setDirectoriesListed(true);
     resourceHandler.setBaseResource(
         Resource.newResource(JettyServerFront.class.getClassLoader().getResource("static")));
     resourceHandler.setWelcomeFiles(new String[] {"index.html"});
-    contextHandler.setHandler(resourceHandler);
-    server.setHandler(contextHandler);
+    return resourceHandler;
+  }
+
+  private static ServletContextHandler addProxyHandler(String dashboardBackAddress) {
+    ProxyServlet proxyServlet = new WebProxyServlet(dashboardBackAddress);
+    ServletHolder holder = new ServletHolder(proxyServlet);
+    holder.setInitParameter("idleTimeout", "120000");
```

Comment:
Could we avoid hard code?

Reply:
> Could we avoid hard code?

Here is redundant, the above on the basis of the Server has set the relevant properties, have been deleted.